addons/mail/static/src/gif_picker/gif_picker.scss

```diff
@@ -0,0 +1,16 @@
+.o-mail-gif-picker {
+    height: 600px;
+    width: 460px;
```

Comment:
I think it can be too high for mobile, the popup can be placed wrong in case of overflow, did you test with mobile UI?